Rasa_Bot/README.md

```diff
@@ -12,18 +12,23 @@ The steps are as follows:
    - The output for the above command should be something like this: [{"recipient_id":"user","text":"Hoi Kees!"},{"recipient_id":"user","text":"Sure, you should ...
    - See [this page](https://rasa.com/docs/rasa/connectors/your-own-website#restinput) for details on how to use the REST channel.
 
+Note that while the requirements-file lists Rasa 2.0.2 as a requirement, this is only needed to train a language model.
+
 ## Future Changes
 ### Language
 Currently, the NLU-model does not use any pre-trained embeddings. If in the future we want to recognize named entities, it might be useful to add such pre-trained embeddings, e.g. via Spacy. More information is provided [here](https://rasa.com/docs/rasa/tuning-your-model). Note that using Spacy requires installing spacy as well as the specific embeddings, e.g. "nl_core_news_lg."
 
+### Rasa Version
+Currently, the model is trained in Rasa 2.0.2. Different Rasa versions are not necessarily compatible w.r.t. e.g. layout of the language model files, so we should eventually choose a specific Rasa version, probably the most current one (2.6.0 at the time of this writing). See [here](https://rasa.com/docs/rasa/changelog) for the changelog for Rasa Open Source.
```

Comment:
Instead of putting this in the Readme, it would be nicer to create an issue for it, so that we also take action on it at some point.